### PR TITLE
client: Improve menumanager

### PR DESCRIPTION
**Pull request by Lucwousin**
Makes swaps work all the time, even with really low fps (lol) or when using mousekeys/anything

### CODE REVIEW

**Comment by tomcylke** [comment]
Doesn't it work the way it does rn for performance reasons? Iirc we had the capability to do this a while ago but we took it out.

**Comment by Lucwousin** [comment]
No it was the way it was because menumanager is a pile of yikes and nobody knew why it was really happening or how to fix it. Come to think of it, this can be improved (easily (as easy as changing 3 lines)) so let me do that real quick

**Comment by tomcylke** [comment]
I stand corrected

**Comment by Lucwousin** [comment]
Seems to have broken for me..? smh

File: runescape-api/src/main/java/net/runelite/rs/api/RSMenuAction.java [hunk 1]
```diff
@@ -24,6 +24,22 @@
...
+	void setOpcode(int yes);
+
+	@Import("argument0")
+	void setIdentifier(int yes);
+
+	@Import("argument1")
+	void setArg1(int yes);
+
+	@Import("argument2")
+	void setArg2(int yes);
```
Comment (Ganom) [inline runescape-api/src/main/java/net/runelite/rs/api/RSMenuAction.java]: Current naming scheme is param0, and param1. Is there a purpose for these being different?

Comment (Lucwousin) [reply runescape-api/src/main/java/net/runelite/rs/api/RSMenuAction.java]: Cause they already were like this and changing is effort, fee; free to do tho

**Comment by Switchssb** [comment]
noob question, but how do i reverse this commit in my live build

**Comment by Crystalknoct** [comment]
Search in Intellij, Version control, Log tab. 
https://i.imgur.com/S2PjO42.jpg
Commit: bfa3b22be6db0f005b0c1364961919b99e19d2b4

**Pull request merged**